code/modules/jobs/job_exp.dm

```diff
@@ -25,7 +29,10 @@ GLOBAL_PROTECT(exp_to_update)
 	body += C.get_exp_report()
 	body += "<A href='?_src_=holder;toggleexempt=\ref[C]'>Toggle Exempt status</a>"
 	body += "</BODY></HTML>"
-	usr << browse(body.Join(), "window=playerplaytime[C.ckey];size=550x615")
+
+	var/datum/browser/popup = new(usr, "playerplaytime[C.ckey]", "Player Playtime",550,615)
+	popup.set_content(body.Join())
+	popup.open()
```

Comment:
Sorry, I'm going to backtrack on this. Admin stuff should be no frills. just use src << browse()